src/logger.test.js

```diff
@@ -0,0 +1,13 @@
+const logger = require("./logger");
+
+describe("The logger", () => {
+  it("should middlware use next", () => {
```

Comment:
I don't understand what does this test mean from its name. You should give name for test, which will indicate, what the function does, and which response it should have

Reply:
Okay, I kind of understand and I renamed test.